packages/frontend-2/components/viewer/anchored-point/Thread.vue

```diff
@@ -32,122 +32,128 @@
-                size="sm"
-                :icon-left="ArrowTopRightOnSquareIcon"
-                text
-                hide-text
-                class="rotate-180"
-                @click="isDragged = false"
-              ></FormButton>
-            </div>
-            <div>
-              <FormButton
-                v-tippy="modelValue.archived ? 'Unresolve' : 'Resolve'"
-                size="sm"
-                :icon-left="
-                  modelValue.archived ? CheckCircleIcon : CheckCircleIconOutlined
-                "
-                text
-                hide-text
-                :color="modelValue.archived ? 'default' : 'default'"
-                :disabled="!canArchiveOrUnarchive"
-                @click="toggleCommentResolvedStatus()"
-              ></FormButton>
-              <FormButton
-                v-tippy="'Copy link'"
-                size="sm"
-                :icon-left="LinkIcon"
-                text
-                hide-text
-                @click="onCopyLink"
-              ></FormButton>
-              <FormButton
-                size="sm"
-                :icon-left="XMarkIcon"
-                text
-                hide-text
-                @click="changeExpanded(false)"
-              ></FormButton>
-            </div>
-          </div>
-          <div class="relative w-80 flex flex-col">
+          <div
+            :class="[
+              'bg-foundation sm:bg-white/70 dark:sm:bg-neutral-800/70 sm:backdrop-blur flex flex-col overflow-hidden sm:shadow-md cursor-auto sm:rounded-lg h-full transition-all duration-200',
+              'group-hover:bg-foundation dark:group-hover:bg-neutral-800 group-[.is-dragging]:bg-foundation dark:group-[.is-dragging]:bg-neutral-800'
+            ]"
+          >
             <div
-              ref="commentsContainer"
-              class="max-h-[500px] overflow-y-auto simple-scrollbar flex flex-col space-y-1 pr-1"
+              class="relative w-full sm:w-80 flex py-2 pl-3 pr-2 sm:px-2 bg-foundation-2"
             >
-              <div
-                v-if="!isThreadResourceLoaded"
-                class="pl-3 pr-1 py-1 mt-2 flex items-center justify-between text-xs text-primary bg-primary-muted"
-              >
-                <span>Conversation started in a different version.</span>
+              <div class="flex-grow flex items-center">
+                <span class="sm:hidden text-primary text-sm font-medium">
+                  Discussions
+                </span>
                 <FormButton
-                  v-tippy="'Load thread context'"
-                  size="xs"
+                  v-tippy="'Previous'"
+                  size="sm"
+                  :icon-left="ChevronLeftIcon"
                   text
-                  @click="onLoadThreadContext"
-                >
-                  <ArrowDownCircleIcon class="w-5 h-5" />
-                </FormButton>
+                  hide-text
+                  @click="emit('prev', props.modelValue)"
```

Comment:
I think this existed before already, but remove the `props.` suffix everywhere in the template, cause all props can be referenced directly there.